src/store/Actions.ts

```diff
@@ -1,14 +1,20 @@
 import type { Dispatch } from "react";
 
-import type { UserSessionInfo } from "./State";
+import type { UserSessionInfo, OrgInfo } from "./State";
 
 /**
  * InnerType are internal action types.
  */
 export enum InnerType {
-  // Real Textile APIs
-  StartSignUp = "START_SIGNUP",
-  FinishSignUp = "FINISH_SIGNUP",
+  // SignUp
+  StartSignUp = "START_SIGN_UP",
+  FinishSignUp = "FINISH_SIGN_UP",
+  // SignIn
+  StartSignIn = "START_SIGN_IN",
```

Comment:
Some new actions. The signup and signin actions are actually all (mostly) identical. Signup will fallback to signin if it gets a "exists" error... this has a strange side effect which we'll have to deal with.